### PR TITLE
Add project setup

### LINKED ISSUE #19: [1pt] Choose assets - group task
Choose images, text, icons, fonts.

### LINKED ISSUE #20: [0.5pt] Find external API - group task
Find an API to base the development of the webapp around it. The API should allow you to
  - Get a list of items with a unique item id (or generate the unique id).
  - For a given item, get detailed information about it.
choose an APIs that doesn't need authentication. if you select an API that requires authentication, you should implement it on your own. Also, if you select an API that provides image resources your webapp will be more visual.

### PR DESCRIPTION
**Pull request by Meri-MG**
**In this group project PR were made several changes and implementations:**
1. In this project was set up webpack, Jest, and linters.
2. The home page.
3. The comments popup.
4. The external NASA  API was selected and it shows the list of items on the screen.
5. The Involvement API shows the item likes and comments
6. When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
7. When the user clicks on the "Comments" button, the Comments popup appears.
8. We have counters in all the interfaces that show: number of items, number of comments.
9. The project was done by using Gitflow and Kanban Board.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "eqeqeq": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
```
Comment (mikenath223) [inline .eslintrc.json]: - [x] Please use the exact eslint rules listed [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)

File: src/comments.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { JSDOM } from 'jsdom';
+import { countElements } from './comments.js';
+
+const dom = new JSDOM(`<!DOCTYPE html><body><ul id="addList" class="flcol"></body>`);// eslint-disable-line
```
Comment (mikenath223) [inline src/comments.test.js]: - [x] Kindly remove the eslint disable rule here. Its best to fix the issue in the code.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+const postScores = async (url, data) => {
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+  return response;
+};
```
Comment (mikenath223) [inline src/index.js:25]: - [ ] Please handle fetch request failures.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+}
+
+function displayPopup(img, title, description, id) {
+  const popupDiv = document.createElement('div');
+  popupDiv.classList.add('popupWindow');
+  popupDiv.innerHTML = `
+  <div class="popupContainer">
+   <span><i class="fas fa-times" id="close"></i>
+   </span>
+   <div class="header-popup">
```
Comment (mikenath223) [inline src/index.js:55]: - [ ] Kindly change these and all other regular functions to es6 arrow function syntax as stated in the Javascript best practices guide.

File: src/index.js (hunk 3 above)
Comment (Meri-MG) [reply src/index.js:25]: Could you clarify this? which request failures? Maybe error screenshot can be helpful

Comment (ZeenatLawal) [reply src/index.js:25]: - [x] The issue here has not been properly resolved. To clarify this, what the reviewer means is to write code that'll manage the fetch request failures. e.g If there's no data to fetch what should happen, check this [link](https://www.codegrepper.com/code-examples/javascript/fetch+api+async+await+try+catch) for an example

File: src/index.js (hunk 4 above)
Comment (ZeenatLawal) [reply src/index.js:55]: - [ ] The issue has not been properly resolved

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @Meri-MG and @GintsM, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [ ] _Kindly check the comments above, made under the reviews of the previous code reviewer(s) and stating **not properly resolved**. Kindly take your time to correctly review those issues outlined by the previous code reviewer, and please check to see that the changes are working as intended._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 4 above)
Comment (Meri-MG) [reply src/index.js:55]: According to this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) js best practices do not ask to use arrow functions, using es6 syntax was put in learning objectives only

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi team, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**